src/Api/SecretsManager/Controllers/ProjectsController.cs

```diff
@@ -130,6 +134,10 @@ public async Task<ListResponseModel<BulkDeleteResponseModel>> BulkDeleteAsync([F
         var userId = _userService.GetProperUserId(User).Value;
 
         var results = await _deleteProjectCommand.DeleteProjects(ids, userId);
+
+        var secretIds = results.SelectMany(projTuple => projTuple.Item1?.Secrets?.Select(s => s.Id) ?? Array.Empty<Guid>()).ToList();
+        await _updateSecretCommand.UpdateRevisionDates(secretIds);
```

Comment:
This should be part of the delete command. When a project is deleted, we update the revisions. I don't think there are any scenarios where we don't want to update the revision date?

Reply:
That's a good point, we will always want to update the revision date.

Updated 👍